### DISCUSSION TITLE
Getting error P3006 : Migration `1_init` failed to apply cleanly to the shadow database.

### DISCUSSION THREAD

**Discussion by Devr-pro** [prisma/prisma#18594] [Q&A]
Hey there ! am getting this error when I tried to evolve the schema and run the migrate command `pnpx prisma migrate dev --name new_model` 
![3006](https://user-images.githubusercontent.com/89377107/229298490-8419e7f4-9d54-4114-b325-5aa9927beacc.jpg)

I have migration sql already in the `prisma/migrations/1_init` as told in setup

**Comment by ludralph** [comment]
Hi @Devr-pro 👋 

Can you please provide the generated SQL that is being executed?
